module/build.sbt

```diff
@@ -11,12 +12,11 @@ crossScalaVersions := Seq("2.10.6", scalaVersion.value)
 resolvers += "Typesafe repository" at "http://repo.typesafe.com/typesafe/releases/"
 
 libraryDependencies ++= Seq(
-  "com.typesafe.play" %% "play" % "2.4.0" % "provided",
-  "com.typesafe.play" %% "play-ws" % "2.4.0" % "provided",
```

Comment:
Whoops, I lost the `provided` scope here, this was not an intentional change. Consequently I've fixed it with a quick push to master in commit 87d24ce8b.